### PR TITLE
Remove PROTOTYPE comments and fix formatting warnings

### PR DESCRIPTION
**Pull request by AlekseyTs**

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review

File: src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs [hunk 1]
```diff
@@ -17332,10 +17332,10 @@ int I2.P6
                                                  targetFramework: TargetFramework.NetStandardLatest);
             Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
 
-            // PROTOTYPE(DefaultInterfaceImplementation): The wording "accessor not found in interface member" is somewhat misleading
-            //                                            in this scenario. The accessor is there, but cannot be implemented. Prehaps
-            //                                            the message should be adjusted. Should also check diagnostics for an attempt
-            //                                            to implement other sealed members.
+            // https://github.com/dotnet/roslyn/issues/34455: The wording "accessor not found in interface member" is somewhat misleading
+            //                                                in this scenario. The accessor is there, but cannot be implemented. Prehaps
```
Comment (cston) [inline src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs]: >Prehaps [](start = 130, length = 7)

Typo

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review, need a second sign-off

File: src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs (hunk 1 above)
Comment (AlekseyTs) [reply src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs]: > Typo

This is preexisting condition, the entire comment will be removed once the referenced issue is addressed.

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review, need a second sign-off for a very small change

**Pull request merged**